CIScripts/Common/Invoke-UntilSucceeds.ps1

```diff
@@ -0,0 +1,56 @@
+. $PSScriptRoot\Aliases.ps1
+. $PSScriptRoot\Exceptions.ps1
+
+function Invoke-UntilSucceeds {
+    <#
+    .SYNOPSIS
+    Utility wrapper for making sure that the assert is consistently true. It works by retrying
+    the assert every Interval seconds, up to Duration.
+    .PARAMETER ScriptBlock
+    ScriptBlock containing a Pester assertion.
+    .PARAMETER Interval
+    Interval (in seconds) between retries.
+    .PARAMETER Duration
+    Timeout (in seconds).
+    #>
+    Param (
+        [Parameter(Mandatory=$true,
+                   ValueFromPipeline=$true)] [ScriptBlock] $ScriptBlock,
+        [Parameter(Mandatory=$false)] [int] $Interval = 1,
+        [Parameter(Mandatory=$false)] [int] $Duration = 3
```

Comment:
`Duration` should be mandatory, because of the same reasoning as in `Eventually` below.